app/containers/AskQuestion/saga.js

```diff
@@ -19,6 +21,7 @@ import {
 } from 'components/QuestionForm/constants';
 
 import { isAuthorized, isValid } from 'containers/EosioProvider/saga';
+import { searchWorker } from 'containers/Search/saga';
```

Comment:
Let's not reuse the same saga from Search container. Please add separate saga that does search for AskQuestion component. There will be separate actions and reducers for search in ask question component.